frontend/src/metabase/browse/containers/TableBrowser/TableBrowser.jsx

```diff
@@ -31,12 +31,8 @@ const getSchemaName = props => {
   return props.schemaName || props.params.schemaName;
 };
 
-const getReloadInterval = (state, { database }, tables = []) => {
-  if (
-    database &&
-    isSyncInProgress(database) &&
-    tables.some(t => isSyncInProgress(t))
-  ) {
+const getReloadInterval = (state, tables = []) => {
```

Comment:
This is where it crashes E2E tests. `getReloadInterval` is passed parameters from `EntityListLoader` so we need to keep the parameter order in tact.
https://github.com/metabase/metabase/blob/5371bf990734c4d359f151a37e46e43ddf539d5a/frontend/src/metabase/entities/containers/EntityListLoader.jsx#L264

You can denote unused parameter with `_` prefix in this case.

```suggestion
const getReloadInterval = (state, _props, tables = []) => {
```